src/sc_client/session.py

```diff
@@ -17,11 +17,11 @@
 from sc_client.client._executor import Executor
 from sc_client.constants import common
 from sc_client.constants.common import ClientCommand
-from sc_client.constants.numeric import LOGGING_MAX_SIZE, SERVER_ANSWER_CHECK_TIME, SERVER_ESTABLISH_CONNECTION_TIME
+from sc_client.constants.numeric import LOGGING_MAX_SIZE, SERVER_ANSWER_CHECK_TIME, SERVER_ESTABLISH_CONNECTION_TIME, SERVER_RECONNECTION_TIME
 from sc_client.models import Response, ScAddr, ScEvent
 
 logger = logging.getLogger(__name__)
-
+reconnection_enable = True
```

Comment:
What you think about passing this flag as an argument to the _set_connection()_ func? By default it can be set as **False**, what shouldn't break a compatibility with previous versions